org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/launch/ClasspathSeparationTest.groovy

```diff
@@ -245,6 +245,7 @@ class ClasspathSeparationTest extends SwtBotSpecification {
     private void launchAndWait(ILaunchConfiguration configuration) {
         ILaunch launch = configuration.launch(ILaunchManager.RUN_MODE, new NullProgressMonitor(), true)
         waitFor { launch.terminated }
+        waitFor { !consoles.activeConsoleContent.isEmpty() }
```

Comment:
Why do we need this?

Reply:
I described the change in the commit message.

Reply:
Sorry, I failed at my own game :)

Reply:
No, I am the one who usually omits the elaboration from the commit message.